dynamo/plot/networks.py

```diff
@@ -1,4 +1,8 @@
+from matplotlib.axes import Axes
 import numpy as np
+import networkx as nx
```

Comment:
do we have networkx as our dependency?

Reply:
Yes, we have networkx in requirements and pyproject.toml (otherwise checks cannot be passed).

Reply:
I mean in our requirements

Reply:
Yes, there is networkx in the requirements.txt.